controller/src/beerocks/master/son_management.cpp

```diff
@@ -2308,6 +2308,95 @@ void son_management::handle_bml_message(Socket *sd,
         LOG(TRACE) << "ACTION_BML_CHANNEL_SCAN_DUMP_RESULTS_REQUEST";
         break;
     }
+    case beerocks_message::ACTION_BML_GET_CLIENT_LIST_REQUEST: {
+        LOG(TRACE) << "ACTION_BML_GET_CLIENT_LIST_REQUEST";
+
+        auto request =
+            beerocks_header->addClass<beerocks_message::cACTION_BML_GET_CLIENT_LIST_REQUEST>();
+        if (!request) {
+            LOG(ERROR) << "addClass cACTION_BML_GET_CLIENT_LIST_REQUEST failed";
+            break;
+        }
+
+        auto response =
+            message_com::create_vs_message<beerocks_message::cACTION_BML_GET_CLIENT_LIST_RESPONSE>(
+                cmdu_tx, beerocks_header->id());
+        if (!response) {
+            LOG(ERROR) << "Failed building message "
+                          "cACTION_BML_GET_CLIENT_LIST_RESPONSE !";
+        }
+
+        //To make sure the response is handled using a faux empty list.
+        auto faux_client_list      = "";
+        auto faux_client_list_size = sizeof(faux_client_list);
```

Comment:
'faux_client_list' is probably a std::string. If I understand what you are trying to do here - you should use `faux_client_list.size()` instead.

Reply:
Changed